api/python/cellxgene_census/src/cellxgene_census/experimental/ml/geneformer_tokenizer.py

```diff
@@ -0,0 +1,154 @@
+import pickle
+from typing import Any, Dict, Optional, Sequence, Set
+
+import numpy as np
+import numpy.typing as npt
+import scipy
+import tiledbsoma
+
+from .cell_dataset_builder import CellDatasetBuilder
+
+GENEFORMER_MAX_INPUT_TOKENS = 2048
+
+
+class GeneformerTokenizer(CellDatasetBuilder):
+    """
+    Generate a Hugging Face `Dataset` containing Geneformer token sequences for each
+    cell in CELLxGENE Census ExperimentAxisQuery results (human).
+
+    This class requires the Geneformer package to be installed separately with:
+    `pip install git+https://huggingface.co/ctheodoris/Geneformer@39ab62e`
+
+    Example usage:
+
+    ```
+    import cellxgene_census
+    import tiledbsoma
+    from cellxgene_census.experimental.ml import GeneformerTokenizer
+
+    with cellxgene_census.open_soma() as census:
+        with GeneformerTokenizer(
+            census["census_data"]["homo_sapiens"],
+            obs_query=tiledbsoma.AxisQuery(...),  # define some subset of Census cells
```

Comment:
Let's provide an actual example with a value_filter. I like filter for cells from tissue == 'tongue' cause it's a very small slice